.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: brew pr-pull
```

Comment:
The workflow files were added automatically when creating an empty skeleton using `brew tap --create`. We'll need to decide if we want to make use of them.

Reply:
We did. They were run with your PR